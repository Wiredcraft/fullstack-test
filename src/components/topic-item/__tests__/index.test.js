import React from 'react';
import renderer from 'react-test-renderer';

import TopicItem from '../index';

jest.mock('react-router-dom');

test('TopicItem renders', async () => {
  const component = renderer.create(<TopicItem />);

  expect(component.toJSON()).toMatchSnapshot();
});
