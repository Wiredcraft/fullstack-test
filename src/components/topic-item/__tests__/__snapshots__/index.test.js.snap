// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopicItem renders 1`] = `
<tr
  className="sc-bdvvtL fuyBFa"
>
  <td
    className="sc-gsDKAQ dEIjVV"
  >
    <div
      className="sc-jRQBWg feKNfK"
    >
      <div
        className="sc-iCfMLu fEMrVh"
        onClick={[Function]}
      />
      <div
        className="sc-furwcr dqEyLy"
        onClick={[Function]}
      />
      <span
        className="sc-gKclnd eeeTcR"
      />
    </div>
  </td>
  <td
    className="sc-dkPtRN DXAXM"
    onClick={[Function]}
  >
    <div
      className="sc-hKwDye ishIcs"
    />
    <div
      className="sc-eCImPb hHGPbi"
    >
      by 
    </div>
  </td>
</tr>
`;
